src/config/dynamic-theme-fixes.config

```diff
@@ -3274,6 +3274,19 @@ CSS
 
 ================================
 
+feedly.com
+
+CSS
+.entry.u0:hover,
+.entry--selected.u0, .entry--selected.u0:hover,
```

Comment:
Every entry on a new line ;)

Reply:
Yes boss! ;) Any reason for this?

Reply:
Code-style :) Just an unified format which I've been trying to 'enforce'. I really should get around and make some documentation on it + some linting rules. So it won't get a mess one day.